solidity/deploy/43_authorize_light_relay_maintainer_proxy_in_light_relay.ts

```diff
@@ -0,0 +1,29 @@
+import { HardhatRuntimeEnvironment } from "hardhat/types"
+import { DeployFunction } from "hardhat-deploy/types"
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  const { getNamedAccounts, deployments } = hre
+  const { execute } = deployments
+  const { deployer } = await getNamedAccounts()
+
+  const LightRelayMaintainerProxy = await deployments.get(
+    "LightRelayMaintainerProxy"
+  )
+
+  // TODO: Find who should call this step
+  await execute(
+    "LightRelay",
+    { from: deployer, log: true, waitConfirmations: 1 },
+    "authorize",
+    LightRelayMaintainerProxy.address
+  )
+}
+
+export default func
+
+func.tags = ["AuthorizeLightRelayMaintainerProxyInLightRelay"]
+func.dependencies = ["LightRelay", "LightRelayMaintainerProxy"]
+
+// TODO: Check if it should be done in mainnet
```

Comment:
We did not transfer the ownership of the `LightRelay` contract during the v2 deployment for this purpose. Let's use the deployer account to authorize the light relay maintainer proxy contract and let's add another step after `93_authorize_relay_maintainer.ts` that will transfer the ownership of the relay to the governance.

Reply:
Removed the TODO in 53bcde8254bdbef80ededa96ff161d8ac99a88da and added ownership transfer of `LightRelay` from `deployer` to `governance` in 01f68f98587bd4c65179c151ead181932055ab86.
The ownership is transferred only for `mainnet`.